projects/fundamental-ngx/src/lib/date-picker/date-picker.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, Input, OnInit, HostListener } from '@angular/core';
+import { DatePickerType } from '../calendar/calendar.component';
+
+@Component({
+    selector: 'fd-date-picker',
+    templateUrl: './date-picker.component.html',
+    styleUrls: ['./date-picker.component.scss']
+})
+export class DatePickerComponent implements OnInit {
+    @Input() type: DatePickerType = 'single';
+    inputFieldDate = null;
+    isValidDateInput: boolean = false;
+    isOpen: boolean = false;
+    dateFromDatePicker: string = '';
+
+    openCalendar(e) {
+        this.isOpen = !this.isOpen;
```

Comment:
If the page has room to scroll, the calendar opens below the input (see screenshot) 
![screen shot 2018-08-15 at 11 12 19 am](https://user-images.githubusercontent.com/2471874/44161997-2ca1f300-a07c-11e8-916e-177330e1be8e.png)

Reply:
will take a look at it!

Reply:
Fixed